recipes/pyaedt/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pyaedt" %}
+{% set version = "0.6.28" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyaedt-{{ version }}.tar.gz
+  sha256: 1fd0c485ecdf43fb506f79964c559b86d9ca908a4272fc11a17c5efb82c75a01
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - mono  # [linux]
+    - dotnet-aspnetcore ==3.1.23  # [linux]
+
+  host:
+    - pip
+    - python
+    - dotnet-aspnetcore ==3.1.23  # [linux]
+    - mono  # [linux]
```

Comment:
It's redundant to have `mono` and `dotnet-aspnetcore` listed in both build and host.

Reply:
Agreed. Thanks for the review.